### PR TITLE
feat: draft cmo schedule online (FPLA-878)

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-647

### Change description ###
Adds the 'Schedule' section to CMO journey

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by Jamesnelson117** [comment]
Branch based off FPLA-3. Blocked until FPLA-3 is merged.

File: ccd-definition/CaseEventToFields/draftCMO.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "cmoHearingDateList",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": 1,
```
Comment (joez-f) [inline ccd-definition/CaseEventToFields/draftCMO.json]: we can have more meaningful pageIDs, sam mentioned to me that they don't have to be 1, 2, 3 etc, so maybe an ID that sums up or describes what the page does - not too important though 👍

Comment (Jamesnelson117) [reply ccd-definition/CaseEventToFields/draftCMO.json]: Yeah I intend to! Need FPLA-3 in before I can polish them off :)

File: e2e/fixtures/schedule.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  allocation: 'The proceedings continue to be allocated to Lewis Birks',
```
Comment (akomisarek) [inline e2e/fixtures/schedule.js]: Is Lewis happy having his real name in the test data? :P

File: e2e/fixtures/schedule.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  application: 'The local authority has applied for a care order',
+  todaysHearing: 'Today\'s case was listed for an INTERIM CARE ORDER HEARING',
```
Comment (akomisarek) [inline e2e/fixtures/schedule.js]: Couldn't we use " and don't escape '? 🤔

File: e2e/fixtures/schedule.js (hunk 2 above)
Comment (lewisbirks) [reply e2e/fixtures/schedule.js]: I'm fine with it

File: e2e/fixtures/schedule.js (hunk 3 above)
Comment (joez-f) [reply e2e/fixtures/schedule.js]: `  4:18  error  Strings must use singlequote  quotes` 😢

Comment (Jamesnelson117) [reply e2e/fixtures/schedule.js]: Yeah unfortunately linter does not like double quotes :/

File: e2e/fixtures/schedule.js (hunk 2 above)
Comment (Jamesnelson117) [reply e2e/fixtures/schedule.js]: Name changed :)

File: e2e/fixtures/schedule.js (hunk 3 above)
Comment (akomisarek) [reply e2e/fixtures/schedule.js]: linter... 👎

File: ccd-definition/CaseEventToFields/draftCMO.json (hunk 1 above)
Comment (akomisarek) [reply ccd-definition/CaseEventToFields/draftCMO.json]: I think numbers are fine for now :)

**Pull request merged**